tiledb/sm/query/reader.cc

```diff
@@ -132,6 +132,42 @@ std::vector<URI> Reader::fragment_uris() const {
   return uris;
 }
 
+Status Reader::get_buffer(
+    const std::string& attribute, void** buffer, uint64_t* buffer_size) const {
+  auto it = attr_buffers_.find(attribute);
+  if (it == attr_buffers_.end()) {
+    *buffer = nullptr;
```

Comment:
Should this branch return an error status?

Reply:
Not necessarily. This means that the buffer has not been set yet, but this is not an error.

Reply:
Oh I see, that is documented on the C API function.